meta-k8s-setup/recipes-k8s-configuration/k8s-configuration/files/k8s-configuration.service

```
@@ -0,0 +1,20 @@
+[Unit]
+Description=service that unpacks the configuration files
+
+After=config.mount
+
+[Service]
+ExecStart=tar -xvf /config/config.tar.gz -C /
+
+TimeoutStartSec=100
+Type=oneshot
+Delegate=yes
+KillMode=process
+Restart=no
+LimitNPROC=infinity
+LimitCORE=infinity
+LimitNOFILE=1048576
+
```

No reason for all this:
```suggestion
Type=oneshot
```